driver/conv_driver.hpp

```diff
@@ -1019,9 +1019,12 @@ int ConvDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
                 break;
             }
 
-            warmup_in  = tensor<warmup_Tgpu>(miopen::deref(warmupInputTensor).GetLengths());
-            warmup_wei = tensor<warmup_Tgpu>(miopen::deref(warmupWeightTensor).GetLengths());
-            warmup_out = tensor<warmup_Tgpu>(miopen::deref(warmupOutputTensor).GetLengths());
+            warmup_in = tensor<warmup_Tgpu>(miopen::deref(warmupInputTensor).GetLengths(),
+                                            miopen::deref(warmupInputTensor).GetStrides());
```

Comment:
Optimization for speed?

Reply:
Nope... this is indeed the way we pass in layout information into `tensor` type, by giving the `stride` information when call constructor.

Reply:
Ah, I see. 

[Recommendation] Revert changes for `warmup_*` tensors. These are always NCHW, and strides can be initialized implicitly.